plugins/kubernetes/app/models/kubernetes/deploy_executor.rb

```diff
@@ -241,6 +241,14 @@ def ensure_build_is_successful(build)
       end
     end
 
+    def rollback(release_docs)
+      release_docs.each do |release_doc|
+        action = release_doc.previous_deploy ? 'Rolling back' : 'Deleting'
+        @output.puts "#{action} #{release_doc.deploy_group.name} role #{release_doc.kubernetes_role.name}"
+        release_doc.revert
+      end
+    end
```

Comment:
At some point we might want to add code to do these in parallel, rather than one at a time